tests/units/signertest.cpp

```diff
@@ -0,0 +1,24 @@
+//#
+//# Copyright (C) 2020-2021 QuasarApp.
+//# Distributed under the lgplv3 software license, see the accompanying
+//# Everyone is permitted to copy and distribute verbatim copies
+//# of this license document, but changing it is not allowed.
+//#
+
+#include "signertest.h"
+
+SignerTest::SignerTest() {
+
+}
+
+SignerTest::~SignerTest() {
+
+}
+
+void SignerTest::test() {
+    testSigner();
+}
+
+void SignerTest::testSigner() {
+    QVERIFY(true);
```

Comment:
Please add warnings message. 
```suggestion
    qWarning() << "The SignerTest class is not implemented"
    QVERIFY(true);
```